packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -320,6 +341,11 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
     }
 
     restoreState(oldState: object): void {
+        // pseudo terminal can not restore
+        if (this.options.isPseudoTerminal) {
```

Comment:
Could we rather avoid storing them to reduce the memory load on localstorage?  it is only 5Mb.